.github/workflows/ci.yml

```diff
@@ -175,7 +175,7 @@ jobs:
   # We only push CI cache as PROD cache usually does not gain as much from fresh cache because
   # it uses prepared airflow and provider packages that invalidate the cache anyway most of the time
   push-early-buildx-cache-to-github-registry:
-    timeout-minutes: 50
+    timeout-minutes: 110
```

Comment:
Nope - it's because in some cases when we rebuild cache from scratch rebuilding cache might take longer - this is just synchronising the timout with the final cache build time at the end of the workflow.